src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Embedded.cs

```diff
@@ -0,0 +1,341 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public class AttributeTests_Embedded : CSharpTestBase
+    {
+        [Fact]
+        public void ReferencingEmbeddedAttributesFromADifferentAssemblyFails_Internal()
```

Comment:
>ReferencingEmbeddedAttributesFromADifferentAssemblyFails_Internal [](start = 20, length = 65)

Should also test with an added module instead of an assembly reference. Should also test if both snippets are compiled into the same source module, there are no errors.